crates/ruff_python_formatter/src/expression/expr_boolean_literal.rs

```diff
@@ -0,0 +1,37 @@
+use ruff_python_ast::AnyNodeRef;
+use ruff_python_ast::ExprBooleanLiteral;
+
+use crate::comments::SourceComment;
+use crate::expression::parentheses::{NeedsParentheses, OptionalParentheses};
+use crate::prelude::*;
+
+#[derive(Default)]
+pub struct FormatExprBooleanLiteral;
+
+impl FormatNodeRule<ExprBooleanLiteral> for FormatExprBooleanLiteral {
+    fn fmt_fields(&self, item: &ExprBooleanLiteral, f: &mut PyFormatter) -> FormatResult<()> {
+        if item.value {
+            token("True").fmt(f)
+        } else {
+            token("False").fmt(f)
+        }
+    }
+
+    fn fmt_dangling_comments(
+        &self,
+        _dangling_comments: &[SourceComment],
+        _f: &mut PyFormatter,
+    ) -> FormatResult<()> {
+        Ok(())
```

Comment:
Nit: Add a debug assertion that `dangling_comments` is empty

```
debug_assert_eq(dangling_comments, &[]);
```

Same for the other literal nodes that can't have dangling comments (all except String, Bytes?)

Reply:
I think it would be the same as not having the `fmt_dangling_comments` method on the nodes, right?